apps/ledger-live-mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -39,6 +39,7 @@
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
         android:windowSoftInputMode="adjustResize"
         android:screenOrientation="portrait"
+        android:exported="true"
```

Comment:
[necessary for targeting API 31](https://stackoverflow.com/questions/70333565/targeting-s-version-31-and-above-requires-that-an-explicit-value-for-android)